data-model/SDP-data-model-templates/person_sdp-template.json

```diff
@@ -16,7 +23,7 @@
     {
```

Comment:
Whether or not we split this out into associated_* (works, persons, places), it strikes me that we may have a mix of manually entered and auto-populated items. For example, it makes the most sense to specify a person as an author of a work _in the work record_, but we may, as with assoc_ms/assoc_witness, want to have the CMS update the person record as well, connecting it to the authored work. I'm not sure this necessarily changes how the data is structured, but would be something to flag to discuss (maybe with t&b?). For instance, I don't know that we'd want a user to be able to manually update a field that has been auto-populated by the CMS (or maybe only certain levels of users would be allowed to?).